### PR TITLE
feat(perf-issues): Add all inclusive resources

### PR DESCRIPTION
**Pull request by 0Calories**
Adds our N+1 docs as a resource that is included across all project platforms. Also removes all third party resources and gets rid of the empty resources state since we always will have at least 1 resource now

### Before
image

### After
image

### CODE REVIEW

File: static/app/components/events/interfaces/performance/resources.tsx [hunk 1]
```diff
@@ -21,21 +21,13 @@ export function Resources(props: Props) {
   return (
     <EventDataSection type="resources-and-whatever" title={t('Resources and Whatever')}>
```
Comment (gggritso) [inline static/app/components/events/interfaces/performance/resources.tsx:22]:
```suggestion
    <EventDataSection type="resources" title={t('Resources')}>
```

now's the time?

**Review by gggritso — APPROVED** [review]
✍🏻

File: static/app/components/events/interfaces/performance/resources.tsx (hunk 1 above)
Comment (0Calories) [reply static/app/components/events/interfaces/performance/resources.tsx:22]: This was an intentional design decision, quirkiness is good. 😎

Comment (0Calories) [reply static/app/components/events/interfaces/performance/resources.tsx:22]: (we can double check with Dora)

Comment (gggritso) [reply static/app/components/events/interfaces/performance/resources.tsx:22]: 😟

**Pull request merged**